assets/prototype/domain/eventEditor/containers/mutations/useUpdatePriceMutation.js

```diff
@@ -1,25 +1,21 @@
 import { useMutation } from '@apollo/react-hooks';
 import { UPDATE_PRICE } from './prices';
-import useToaster from '../../../../infrastructure/services/toaster/useToaster';
+import useInitToaster from '../../../../infrastructure/services/toaster/useInitToaster';
 import useRelations from '../../../../infrastructure/services/relations/useRelations';
 
 const useUpdatePriceMutation = ({ id = 0 }) => {
-	const toaster = useToaster();
 	const { updateRelations } = useRelations();
 	const toasterMessage = `updating price ${id}`;
-	const [updatePrice, { loading, error }] = useMutation(UPDATE_PRICE, {
-		onCompleted: () => {
-			toaster.dismiss(toasterMessage);
-			toaster.success(`price ${id} successfully updated`);
-		},
-		onError: (error) => {
-			toaster.dismiss(toasterMessage);
-			toaster.error(error);
-		},
+
+	const initToaster = useInitToaster({
+		toasterMessage,
+		loadingMessage: toasterMessage,
+		successMessage: `price ${id} successfully updated`
 	});
 
-	toaster.loading(loading, toasterMessage);
-	toaster.error(error);
+	const [updatePrice, { loading, error }] = useMutation(UPDATE_PRICE, initToaster);
+
+	initToaster.initializationNotices(loading, error);
```

Comment:
Did you know I'm a big fan of whiteboards? I find them quite re-markable.

and here...